docs/core/linux-prerequisites.md

```diff
@@ -19,6 +19,32 @@ This article shows the dependencies needed to develop .NET Core applications on
 
 <!-- markdownlint-disable MD025 -->
 
+# [.NET Core 3.0](#tab/netcore30)
+
+.NET Core 3.0 treats Linux as a single operating system. There is a single Linux build (per chip architecture) for supported Linux distributions. 
+
+For download links and more information, see [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+
+.NET Core 3.0 is supported on the following Linux distributions/versions. 
+
+OS                            | Version               | Architectures  
+------------------------------|-----------------------|----------------
+Red Hat Enterprise Linux      | 6                     | x64
+Red Hat Enterprise Linux<br>CentOS<br>Oracle Linux  | 7                     | x64
+Fedora                        | 28, 29, 30            | x64
+Debian                        | 9, 10                 | x64, ARM32\*, ARM64\*
+Ubuntu                        | 16.04+, 18.04+, 19.04+ | x64, ARM32\*, ARM64\*
```

Comment:
If you're going to use the + notation, instead of "or later versions", you need to explain what the + means before the table. Not all international customers understand that concept.

Reply:
There are no more + in the article, so this one is resolved. :)